tests/accordion-test.ts

```diff
@@ -0,0 +1,414 @@
+                </div>
+                <div class="container acc-active">
+                    <div class="acc-title">Title 2</div>
+                    <div class="acc-body acc-active">Content 2</div>
+                </div>
+            </aalam-accordion>
+        `);
+        expect(
+            element.querySelectorAll(".acc-body.acc-active").length
+        ).to.equal(2);
+        expect(element.querySelector(".acc-body.acc-active")).to.exist;
+        expect(element.querySelectorAll(".acc-active")).to.exist;
+    });
+
+    it("allows multiple items to be open if nomultiple y ", async () => {
+        const el = await fixture(html`
+            <aalam-accordion activecls="acc-active">
+                <div class="container ">
+                    <div class="acc-title">
+                        <span>What is Artificial Intelligence?</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">
+                        <p>body 1</p>
+                    </div>
+                </div>
+                <div class="container  ">
+                    <div class="acc-title">
+                        <span>What is Machine Learning?</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 2</div>
+                </div>
+
+                <div class="container  ">
+                    <div class="acc-title">
+                        <span>title 3?</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 3</div>
+                </div>
+
+                <div class="container  ">
+                    <div class="acc-title">
+                        <span>title 4</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 4</div>
+                </div>
+
+                <div class="container  ">
+                    <div class="acc-title">
+                        <span>title 5</span>
+                        <button>Toggle</button>
+                    </div>
+                    <div class="acc-body">body 5</div>
+                </div>
+            </aalam-accordion>
+        `);
+
+        const titles = el.querySelectorAll(".acc-title");
+        titles[0]?.click();
+        titles[1]?.click();
+        await el.updateComplete;
+        expect(el.querySelectorAll(".acc-active").length).to.equal(2);
+
+        titles[0]?.click();
+        titles[3]?.click();
+        titles[4]?.click();
+        await el.updateComplete;
+        expect(el.querySelectorAll(".acc-active").length).to.equal(3);
+
+        titles[1]?.click();
+        titles[3]?.click();
+        titles[4]?.click();
+        await el.updateComplete;
+        expect(el.querySelectorAll(".acc-active").length).to.equal(0);
+
+        const ele = await fixture(html`
```

Comment:
Instead of creating a new element, we cna set the attribute to the old element itself. You can also remove that attribute later and check the behaviour is same as the beginning